build/images/base/build.sh

```diff
@@ -29,13 +29,19 @@ Build the antrea base image.
         --push                  Push the built image to the registry
         --platform <PLATFORM>   Target platform for the image if server is multi-platform capable
         --distro <distro>       Target Linux distribution
-        --no-cache              Do not use the local build cache nor the cached image from the registry"
+        --no-cache              Do not use the local build cache nor the cached image from the registry
+        --build-tag             Custom build tag for images"
```

Comment:
Better to add explicitly --no-cache and --build-tag options as inline options as well ex usage="Usage: $0 [--pull] [--push] [--platform <PLATFORM>] [--distro [ubuntu|ubi]] [--no-cache] [--build-tag <custom-build-tag> ]

Reply:
I didn't find the need for the same, is there any requirement?

Reply:
better wrt user, build/images/base/build.sh --help
Usage: build/images/base/build.sh [--pull] [--push] [--platform <PLATFORM>] [--distro [ubuntu|ubi]] [--no-cache] [--build-tag]
Build the antrea base image.
        --pull                  Always attempt to pull a newer version of the base images
        --push                  Push the built image to the registry
        --platform <PLATFORM>   Target platform for the image if server is multi-platform capable
        --distro <distro>       Target Linux distribution
        --no-cache              Do not use the local build cache nor the cached image from the registry
        --build-tag             Custom build tag for images"

Reply:
--no-cache field was not added by me, it was added by other developer, need to check why wasn't it added then.

Reply:
Let's add it for --no-cache as well